src/components/Page/Page.js

```diff
@@ -28,6 +30,11 @@ export const Page = ({
     >
         <header className={styles.header}>
             <h1 className={styles.title}>{title}</h1>
+            {version && (
+                <Tag className={styles.tagVersion}>
+                    {i18n.t('v{{version}}', { version: version })}
```

Comment:
I'm curious why the `i18n` translation is needed and if the version is different in different languages? Thanks!

Reply:
Currently it is a "just in case" translation. But if it's better to remove the translation I can do that.

Reply:
That's up to you. Either way it seems fine to me. I was just curious if a particular use case already existed